.github/workflows/test-electron.yml

```diff
@@ -8,7 +8,7 @@ jobs:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        electron: [ '^11.2.0' ]
+        electron: [ '^12.0.0', '^20.0.0' ]
         node-version: [12, 14]
         os: [ ubuntu-20.04, windows-2019, macos-10.15 ]
```

Comment:
it looks like macos-10.15 will soon be out of support and is experiencing brownouts, is this a good time to update it?

@joshedney

Reply:
Yeah. I opened the PR before I saw the bit about brownouts.